flink-end-to-end-tests/flink-connector-gcp-pubsub-emulator-tests/src/test/java/org/apache/flink/streaming/connectors/gcp/pubsub/emulator/GCloudEmulatorManager.java

```diff
@@ -207,7 +207,7 @@ private static boolean waitForOkStatus(String label, String port) {
 	private static String getPort(Map<String, List<PortBinding>> ports, String internalTCPPort, String label) {
 		List<PortBinding> portMappings = ports.get(internalTCPPort + "/tcp");
 		if (portMappings == null || portMappings.isEmpty()) {
-			LOG.info("| {} Emulator {} --> NOTHING CONNECTED TO {}", label, internalTCPPort + "/tcp");
+			LOG.info("| {} Emulator --> NOTHING CONNECTED TO {}/tcp", label, internalTCPPort);
```

Comment:
I can't shake the feeling that I reviewed this change already in another PR...

Reply:
Emm... Maybe another PR changed this line conveniently, could you find that PR due to the master code still has this error.
Anyway, this PR mainly focus on correcting the argument mismatching and I should find out all wrong places.